python/lsst/ts/genericcamera/driver/alliedvisioncamera.py

```diff
@@ -0,0 +1,302 @@
+# This file is part of ts_genericcamera.
+#
+# Developed for the Vera Rubin Observatory Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import asyncio
+import concurrent.futures
+import functools
+
+import vimba
+import yaml
+
+from . import basecamera
+from .. import exposure
+from ..fits_header_items_generator import FitsHeaderItemsGenerator, FitsHeaderTemplate
+
+MILLISECONDS = 10**3
```

Comment:
Instead of expressing this as a math computation, why not use the number directly? e.g. `1e3`? It avoids one expensive computation. If you want it to be an internet, I'd do `1000` instead of `10**3`.

Also, is this supposed to be the scale in seconds (`1ms = 1e-3s`)? If so, shouldn't it be `1e-3` instead? If it is the reverse, please, clarify.

Reply:
Fixed the short-hand notation to just use numbers and changed the variable names to hopefully make them more understandable.